olp-cpp-sdk-authentication/include/olp/authentication/Types.h

```diff
@@ -59,6 +58,7 @@ using UserAccountInfoResponse = Response<model::UserAccountInfoResponse>;
 using UserAccountInfoCallback = Callback<model::UserAccountInfoResponse>;
 
 /// The access token response type.
+class TokenResult;
 using TokenResponse = Response<TokenResult>;
```

Comment:
Moving the TokenResult to private makes this impossible to use for any outside library linked against Data SDK C++. Please remove.

Reply:
removed, added header